### PR TITLE
Translate emails

### PR DESCRIPTION
**Pull request by andnuneCloudmanLabs**
Added internacionalization for emails.

### CODE REVIEW

**Comment by snipe** [comment]
Thanks again for this! I'm showing a merge conflict here. Can you pull down the latest from develop and resolve any conflicts? Would love to get this merged in.

**Comment by andnuneCloudmanLabs** [comment]
Conflicts resolved.

File: app/Http/Controllers/Auth/PasswordController.php [hunk 1]
```diff
@@ -35,6 +35,7 @@ class PasswordController extends Controller
      */
     public function __construct()
     {
+        $this->subject = \Lang::get('mail.reset_link');
```
Comment (dmeltzer) [inline app/Http/Controllers/Auth/PasswordController.php]: This looks wrong, PasswordController shouldn't have a subject, right?

File: config/database.php [hunk 2]
```diff
@@ -61,8 +61,8 @@
         'mysql' => [
             'driver'    => 'mysql',
             'host'      => env('DB_HOST', 'localhost'),
-            'database'  => env('DB_DATABASE', 'forge'),
-            'username'  => env('DB_USERNAME', 'forge'),
+            'database'  => env('DB_DATABASE', ''),
+            'username'  => env('DB_USERNAME', ''),
```
Comment (dmeltzer) [inline config/database.php]: Changes to configs probably shouldn't be part of this PR.

**Comment by andnuneCloudmanLabs** [comment]
Yes, you're right, now it overrides getEmailSubject function from "ResetPasswords" trait, this way we can translate email subject. Changes to config was just a mistake.
